### PR TITLE
Get involved page 3  roadmap/ memorandum  UI

### PR DESCRIPTION
**Pull request by NayoBaez**
**Get Involved Page**
_Assigned issues:  #48 & #108_  
- Added all included links to text & buttons 
- Double checked all instances of "Orcasound" 
- Provided controls to allow user to zoom in / out / reset image
- Made responsive to desktop / tablet & mobile as in [Figma Design](https://www.figma.com/file/YdvbAjHETokV2x1LBHaElj/Orcahome-%2F-Orcasound-Website-Redesign-2021?type=design&node-id=860-2526&t=qGFRynj9BuoKUioI-0#409937693)

### CODE REVIEW

**Comment by JacobYLiu** [comment]
@evanjscallan We finished fixing the checks on this pull request as well.

File: package.json [hunk 1]
```diff
@@ -22,11 +22,14 @@
     "@emotion/styled": "^11.6.0",
     "@mui/icons-material": "^5.3.1",
     "@mui/material": "^5.3.1",
-    "next": "^12.0.9",
+    "audit": "^0.0.6",
+    "next": "^12.3.4",
```
Comment (evanjscallan) [inline package.json:26]: Not sure if the next version should change here.

File: src/pages/getinvolved.jsx [hunk 2]
```diff
@@ -209,66 +213,258 @@ export const GetInvolved = () => {
...
+                </Box>
+                <TransformComponent>
+                  <Image src={roadmap} alt="roadmap" />
+                </TransformComponent>
+              </>
+            )}
+          </TransformWrapper>
+          <Typography
+            variant="p"
+            fontSize="16px"
```
Comment (evanjscallan) [inline src/pages/getinvolved.jsx:272]: seeing fontSize of 16px and 20px which are fairly similar. Not sure if intention

File: src/pages/getinvolved.jsx [hunk 3]
```diff
@@ -209,66 +213,258 @@ export const GetInvolved = () => {
...
+          free!) either as the host of an hydrophone node, a researcher or
+          citizen scientist an educator/activist or general volunteer.
+        </Typography>
+        <Typography
+          variant="p"
+          fontSize="20px"
+          paragraph={true}
+          align="justify"
+        >
+          If you are an individual wanting to volunteer , collaborate or donate,
```
Comment (evanjscallan) [inline src/pages/getinvolved.jsx:395]: comma spacing

File: src/pages/getinvolved.jsx [hunk 4]
```diff
@@ -209,66 +213,258 @@ export const GetInvolved = () => {
...
+          a new hydrophone node, an educational/outreach node, or both -- just
+          read the history, mission and vision of the network e-sign the MOA and
+          then email{' '}
+          <Link
+            href="mailto:info@orcasound.net"
+            style={{ textDecoration: 'none', color: '#1B2B7B' }}
+          >
+            info@orcasound.net
+          </Link>
+          {''} to begin collaborating. There are no membership fees-- just
```
Comment (evanjscallan) [inline src/pages/getinvolved.jsx:415]: spacing

File: package.json [hunk 5]
```diff
@@ -22,11 +22,14 @@
...
+    "next-image-zoom": "^1.1.7",
```
Comment (evanjscallan) [inline package.json:27]: where is this being used?

**Review by evanjscallan — CHANGES_REQUESTED** [review]
See comments

File: src/pages/getinvolved.jsx (hunk 3 above)
Comment (NayoBaez) [reply src/pages/getinvolved.jsx:395]: Fixed, thank you!

File: src/pages/getinvolved.jsx (hunk 4 above)
Comment (NayoBaez) [reply src/pages/getinvolved.jsx:415]: Fixed, thank you!

**Comment by NayoBaez** [comment]
@evanjscallan Hi Evan, how are you? Just wanted your guidance, i fixed all the review changes but now my checks won't pass. It's asking me to update the typescript version and also reinstall the latest Netlify plugin version. If i update they won't match with the original versions. What would you recommend?

**Comment by UXBrendan** [comment]
@UXBrendan 

Deploy preview passes UI check. 😎

**Review by scottveirs — APPROVED** [review]
Thanks for responding to Evan's feedback. Looks good to the oceanographer!

**Pull request merged**